speed-dreams/source-2.2.3/src/modules/sound/snddefault/OpenalSoundInterface.cpp

```diff
@@ -219,11 +219,12 @@ Sound* OpenalSoundInterface::addSample (const char* filename, int flags, bool lo
 /// @brief Updates all sounds related to interventions. Makes sure the right ones are playing and the right ones are stopped.
 /// @param p_carSoundData Data related to the car, like position data.
 /// @param p_interventionSounds The registered sounds
```

Comment:
No param p_interventionSounds anymore
```suggestion
/// @brief Updates all sounds related to interventions. Makes sure the right ones are playing and the right ones are stopped.
/// @param p_carSoundData Data related to the car, like position data.
```